apps/explorer/src/modules/blocks/components/TableBlocks.vue

```diff
@@ -116,6 +95,8 @@ import AppPaginate from '@app/core/components/ui/AppPaginate.vue'
 import AppPaginateHasMore from '@app/core/components/ui/AppPaginateHasMore.vue'
 import AppTableTitle from '@app/core/components/ui/AppTableTitle.vue'
 import TableBlocksRow from '@app/modules/blocks/components/TableBlocksRow.vue'
+import TableBlocksRowLoading from '@app/modules/blocks/components/TableBlocksRowLoading.vue'
+
```

Comment:
Remove extra space

Reply:
will do

Reply:
fixed in  [e2c9d...](https://github.com/EthVM/EthVM/pull/729/commits/e2ec9d4c8201776419606cf0a623f38c119411ac)